src/client.test.js

```diff
@@ -122,7 +126,8 @@ describe('Client side', () => {
       </ContextProvider>
     )
 
-    expect(component.html()).toBe('<div><div>No errors! Context variable</div></div>')
+    expect(component.html()).toBe('<div>No errors! Context variable</div>')
```

Comment:
Keeping these expectations to show things passing, but it'd be awesome to switch solely to snapshots.